nipype/info.py

```diff
@@ -141,7 +141,8 @@ def get_nipype_gitversion():
     'configparser',
     'pytest>=%s' % PYTEST_MIN_VERSION,
     'mock',
-    'pydotplus'
+    'pydotplus',
+    'packaging',
```

Comment:
Right. I don't think `packaging` is required by that. Running `python setup.py bdist_wheel` worked fine. I then also ran `python setup.py install` which required the other dependencies.

Sorry if I'm missing something.